src/rosenbrock.jl

```diff
@@ -0,0 +1,89 @@
+#ODEROSENBROCK Solve stiff differential equations, Rosenbrock method
```

Comment:
Because nothing changed with them. Then leaving them in the place they were, the diff will show that indeed nothing changed.  This is a huge PR already, so anything stable is appreciated as that is less to review.  Same goes for the Adams solvers.  Once they get updated (which should be a separate PR) then I'm all for it to have them in a separate file.